cmd/kubelet/app/server.go

```diff
@@ -502,6 +502,20 @@ func InitializeTLS(kc *componentconfig.KubeletConfiguration) (*server.TLSOptions
 		CertFile: kc.TLSCertFile,
 		KeyFile:  kc.TLSPrivateKeyFile,
 	}
+
+	if ks.TLSCertAuth {
```

Comment:
Actually I wanted to implement this in easy way. Using interfaces will add complexity, because you'll need to check lot of stuff and add fake authorizer functions. And most probably this fix would not be ported back to 1.3.x. So I'd suggest to keep it as is if you don't mind. More advanced authorizer function could be implemented in separate PR.
What do you think?